src/ceph-daemon/ceph-daemon

```diff
@@ -1616,6 +1616,105 @@ def command_rm_cluster():
 
 ##################################
 
+def check_time_sync():
+    units = ['chronyd.service', 'systemd.timesyncd', 'ntpd.service']
+    for u in units:
+        (enabled, state) = check_unit(u)
+        if enabled and state == 'running':
+            logger.info('Time sync unit %s is enabled and running' % u)
```

Comment:
should new code use the older % style formatting?

Reply:
I personally don't care about `%` or `.format` as long as we stick to f-strings in py3 only code.